<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordMainWindow</class>
 <widget class="QMainWindow" name="RecordMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>868</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Webrtc推流demo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget, QLabel{
                        color:white;
                        border: 1px solid white;
                        padding:0;
                        border-radius:5px;
							background-color:rgb(20,50,70);
                      /**  background-color:rgba(250,170,0,150);**/
  }

/*TabWidget*/
QTabWidget{
/*给tabwidget设置etAttribute(Qt::WA_StyledBackground)*/
border: 1px solid white;
background-color:rgb(20,50,70);
}
QTabWidget::tab-bar{
alignment:left;
left:1px;
}
QTabWidget::pane { 
/*background-color:rgb(20,50,70);*/
border:1px solid white;
} 
QTabBar{
/*只有有按钮的地方才是tabbar，空白处是tabwidget的*/
background-color:white;
}
QTabBar::tab{/*页签*/
min-height:28px;
padding:0 10px;
border:0;
margin:1px 1px 0 0;
background-color:rgb(20,50,70);
}
QTabBar::tab:first{
margin-left:1px;
}
QTabBar::tab:hover{
color:cyan;
}
QTabBar::tab:selected{
background-color: rgb(20, 100, 150);
}
QTabBar::tab:selected:hover{
}

QTabBar::tear{
}
QTabBar::scroller{
}
QMenu {
     background-color: rgb(20,50,70); /* sets background of the menu 设置整个菜单区域的背景色，我用的是白色：white*/
     border: 1px solid white;/*整个菜单区域的边框粗细、样式、颜色*/
 }
  
 QMenu::item {
     /* sets background of menu item. set this to something non-transparent
         if you want menu color and menu item color to be different */
     background-color: transparent;
     padding:8px 32px;/*设置菜单项文字上下和左右的内边距，效果就是菜单中的条目左右上下有了间隔*/
     margin:0px 8px;/*设置菜单项的外边距*/
     border-bottom:1px solid #DBDBDB;/*为菜单项之间添加横线间隔*/
 }
  
 QMenu::item:selected { /* when user selects item using mouse or keyboard */
     background-color: #2dabf9;/*这一句是设置菜单项鼠标经过选中的样式*/
 }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="m_b_rec">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>660</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>推流</string>
    </property>
   </widget>
   <widget class="QFrame" name="vdMain">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1081</width>
      <height>601</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="m_c_screen">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>660</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>屏幕</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_l_url">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>660</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>URL:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_url">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>660</y>
      <width>601</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_t_data">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>713</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>hello,metaRTC5.0!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_b_senddc">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>710</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Senddata</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_l_recv">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>710</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="m_l_url_2">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>710</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>RECV:</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_l_err">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>770</y>
      <width>861</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="m_l_url_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>770</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Error:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
